.gitattributes

```diff
@@ -1 +1,8 @@
 src/clientversion.cpp export-subst
+
+# Ensures Windows pulls line ending dependent files correctly for Makefile's build process:
+depends/Makefile text eol=lf
```

Comment:
If it works by explicitly include for all files, a wildcard should do the work.

Replacing all of those lines by `*    text    eol=lf` should be equivalent, forcing the entire repo to use always use LF and avoid CRLF line endings.

Reply:
We'll probably want to wildcard only files being built in the makefile. I wouldn't want to LF the visual studio solution, for example.